http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -92,7 +92,13 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
             case Some(cn) =>
               cn match {
                 case AllowedChartName.RStudio => F.unit
-                case AllowedChartName.Sas     => F.unit // TODO: check group permission check here
+                case AllowedChartName.Sas =>
+                  if (config.enableSasAppGroupCheck)
+                    authProvider.isSasAppAllowed(userInfo.userEmail) map { res =>
```

Comment:
Would that work while we support multi-cloud(e.g. using MSFT account)/RAS as primary login.
Can we use use GCP project security group labels instead?  i.e. SAS can only run on AoU powered projects?

Reply:
it should...it's a Sam group. We may support SAS on Terra soon as well